src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/blocks/needbraces/Example1.java

```diff
@@ -0,0 +1,53 @@
+/*xml
+<module name="Checker">
+  <module name="TreeWalker">
+    <module name="NeedBraces"/>
+  </module>
+</module>
+*/
+
+package com.puppycrawl.tools.checkstyle.checks.blocks.needbraces;
+
+// xdoc section -- start
+class Example1 {
+  String obj = new String();
+  public boolean Example1() {
+    int num = 12;
+    if (obj.equals(num)) return true;
+    // violation above, ''if' construct must use '{}'s.'
+    // single-line statements not allowed without braces.
```

Comment:
As we becomes to show violation message, `single-line statements not ...` is not required and even confuse. Please remove it in all example files.

Reply:
done !